README.md

```diff
@@ -35,18 +33,12 @@ gradle build
 cp src/main/resources/config.sample.json build/resources/main/config.json
 sed -i 's/"secret_key": "",/"secret_key": "thisisjustavalueandnotarealsecretsomemorecharactersuntilwehave64",/' ./build/resources/main/config.json
 ```
-5. Create and configure front end
+5. Create front end
 ```bash
 ( cd webapp/
 yarn install
 ./build.sh nl
-cat > build/assets/config.js <<EOD
-var config = {
-  IRMASERVER: 'http://localhost:8088',
-  EMAILSERVER: 'http://localhost:8080/irma_email_issuer',
-};
-EOD
-)
+cp webapp/config.example.js build/assets/config.js
 cp -a webapp/ src/main/
```

Comment:
I think that final step is not needed to copy webapp to src/main, because the Java-part does not need it.

Reply:
```
$ gradle appRun

> Task :appRun
You're running webapp in hard inplaceMode: Overlay and filtering features of gretty will be disabled!

11:21:35 WARN  Web application not found /tmp/irma_email_issuer/src/main/webapp
11:21:35 WARN  Failed startup of context o.a.g.JettyWebAppContext@5a709816{/irma_email_issuer,null,null}{/tmp/irma_email_issuer/src/main/webapp}
java.io.FileNotFoundException: /tmp/irma_email_issuer/src/main/webapp
```